src/test/ui/parser/suggest-const-for-global-var.rs

```diff
@@ -0,0 +1,8 @@
+// run-rustfix
+
+let X: i32 = 12;
```

Comment:
what if this is a function call that is not a `const fn` ?
`let X: i32 = get_random_number();`

Reply:
> what if this is a function call that is not a `const fn` ? `let X: i32 = get_random_number();`

Yes, if we replace `let` with `instead` in this case, it will trigger another error:

```console
error[E0015]: cannot call non-const fn `get_random_number` in constants
 --> src/test/ui/parser/suggest-const-for-global-var.rs:7:16
  |
7 | const X: i32 = get_random_number();
  |                ^^^^^^^^^^^^^^^^^^^
  |
  = note: calls in constants are limited to constant functions, tuple structs and tuple variants

error: aborting due to previous error
```

You mean we should not use 'Applicability::MachineApplicable' in suggestion?
You remind me that top level `let` may also be replaced with `static`, I will make a change for this.

`const` and `static` are both limited to constant functions, tuple structs and tuple variants.